static.config.json

```diff
@@ -3,7 +3,7 @@
     "apiHost": "http://localhost:4001/dev",
     "cmrHost": "http://localhost:4000",
     "env": "development",
-    "gkrHost": "https://gkr.sit.earthdatacloud.nasa.gov",
+    "gkrHost": "https://gkr.earthdatacloud.nasa.gov",
```

Comment:
Does deploy_bamboo.sh update this?  If so, did you verify it is updating it correctly?